app/containers/Avatar/Avatar.tsx

```diff
@@ -80,7 +81,11 @@ const Avatar = React.memo(
 		}
 
 		if (onPress) {
-			image = <Touchable onPress={onPress}>{image}</Touchable>;
+			image = (
+				<Touchable accessibilityLabel={accessibilityLabel} onPress={onPress}>
```

Comment:
Here you're adding accessibility label only if avatar has onPress.
You should add it to that view below if there's no onPress.

You can test that by checking your own avatar on RoomView.
You'll see it's not touchable, but it needs to be read.

Reply:
got it, adjusted.